vue-model-api/build.gradle.kts

```diff
@@ -47,13 +47,11 @@ tasks.named("npm_run_lint") {
     outputs.cacheIf { true }
 }
 
+val packageJsonForProd = layout.buildDirectory.file("package-for-publishing.json").get().asFile
 val createPackageJsonForPublishing = tasks.create("createPackageJsonForPublishing") {
     dependsOn(updateModelClient)
-    val packageJsonForProd = projectDir.resolve("dist/package.json")
     val packageJsonForDev = projectDir.resolve("package.json")
-    inputs.file(packageJsonForDev)
-    outputs.cacheIf { true }
-    outputs.file(packageJsonForProd)
+    // This task depends on `project.version` and can therefore not be cached.
```

Comment:
That would be possible in general. 

- This task only creates one file (Is the overhead of caching bigger than creating it?)
- The version changes every on every publish workflow.
- But for local development, this would bring us to builds that only execute tasks, that really changed.

Let me enable caching again and make the version an input.

Reply:
Done. Can be reviewed again.